quartz-plugins/src/main/java/org/quartz/plugins/interrupt/PluginConstants.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2001-2009 Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.quartz.plugins.interrupt;
+
+/**
+ * The Class PluginConstants.
+ * 
+ * @author Rama Chavali
+ */
+public class PluginConstants {
```

Comment:
Is there a good reason to have these constants defined on a different class than the plugin class itself?

My preference would be for them to be on the plugin class.  (that matches other plugins).   Or at least the name of this class should be more complete (i.e. prefixed with JobInterruptMonitor) - and perhaps it should even be an interface.